From bdaa45fb330b085e5adaea11bc3d1bb18d100cc3 Mon Sep 17 00:00:00 2001
From: Samantha N. Bueno <sbueno+anaconda@redhat.com>
Date: Mon, 06 May 2013 18:40:53 +0000
Subject: Fix a minor display issue in TUI.

Always set completed status to True in the checkbox widgets of the
spoke display, this way the key is always visible.
---
diff --git a/pyanaconda/ui/tui/spokes/__init__.py b/pyanaconda/ui/tui/spokes/__init__.py
index b770351..134e303 100644
--- a/pyanaconda/ui/tui/spokes/__init__.py
+++ b/pyanaconda/ui/tui/spokes/__init__.py
@@ -78,7 +78,9 @@ class TUISpoke(TUIObject, tui.Widget, Spoke):
             key = "x"
             completed = self.completed
 
-        c = tui.CheckboxWidget(key = key, completed = completed,
+        # always set completed = True here; otherwise key value won't be
+        # displayed if completed (spoke value from above) is False
+        c = tui.CheckboxWidget(key = key, completed = True,
                                title = self.title, text = self.status)
         c.render(width)
         self.draw(c)
--
cgit v0.9.1
